src/graph/CreateEdgeIndexExecutor.cpp

```diff
@@ -31,14 +31,24 @@ void CreateEdgeIndexExecutor::execute() {
     auto *edgeName = sentence_->edgeName();
     auto columns = sentence_->names();
     auto spaceId = ectx()->rctx()->session()->space();
+    if (UNLIKELY(columns.empty())) {
```

Comment:
The check about column's empty and column duplicate is in meta.

Reply:
That's my purpose.